README.md

```diff
@@ -1,107 +1,108 @@
-* 👩‍🔬 **ML Researchers and Data Scientists** who don't want to spend or wait 3-6 months translating and packaging
-their work for production.
-* 👩‍🏭 **ML Engineers** who want to be able to update and improve production services, pipelines, and artifacts with a
-Pythonic, debuggable devX.
-* 👩‍🔧 **ML Platform teams** who want a versioned, shared, maintainable stack of services and data artifacts that
-research and production pipelines both depend on.
-
-### 🦾 How does it work?
-
-_"Learn once, run anywhere"_
+Runhouse is a Python framework for composing and sharing production-quality backend apps and services _ridiculously
+quickly_ on any infra. Running the following will stand up a microservice on a fresh AWS EC2 box:
 
-Runhouse is like **PyTorch + Terraform + Google Drive.**
+```python
+import runhouse as rh
 
-1. Just as **PyTorch** lets you send a model or tensor `.to(device)`, Runhouse OSS
-lets you do `my_fn.to('gcp_a100')` or `my_table.to('s3')`: send functions and data to any of your compute or
-data infra, all in Python, and continue to interact with them eagerly (there's no DAG) from your existing code and
-environment. Think of it as an expansion pack to Python that lets it take detours to remote
-machines or manipulate remote data.
-2. Just as **Terraform** is a unified language for creation and destruction of infra, the
-Runhouse APIs are a unified interface into existing compute and data systems.
-See what we already support and what's on the [roadmap, below](#supported-infra).
-3. Runhouse resources can be shared across environments or teams, providing a **Google Drive**-like
-layer for accessibility, visibility, and management across all your infra and providers.
+def run_home(name: str):
+    return f"Run home {name}!"
 
-This allows you to:
-* Call your preprocessing, training, and inference each on different hardware from
-inside a single notebook or script
-* Slot that script into a single orchestrator node rather than translate it into an ML pipeline DAG of Docker images
-* Share any of those services or data artifacts with your team instantly, and update them over time
+if __name__ == "__main__":
+    cpu_box = rh.ondemand_cluster(name="my-cpu", instance_type="CPU:2", provider="aws")
+    remote_fn = rh.function(run_home).to(cpu_box)
+    print(remote_fn("Jack"))
+```
 
-![img.png](https://raw.githubusercontent.com/run-house/runhouse/main/docs/assets/img.png)
-![img.png](https://raw.githubusercontent.com/run-house/runhouse/main/docs/assets/img_1.png)
+This is regular Python running on your own machine(s) with your existing code. There's no YAML, no magic CLI
+incantations, no decorators, and no prior setup (other than `pip install runhouse` and having `~/.aws/credentials` in
+this case).
+
+Runhouse is built to do four things:
+1. Bring the power of Ray to any app, anywhere, without having to learn Ray or manage Ray clusters, like Next.js did
```

Comment:
There are users who will have no idea what Ray is? I don't think this should be the first thing we list here

Reply:
Great point